pytorch_lightning/overrides/fairscale.py

```diff
@@ -29,3 +29,22 @@ def unwrap_lightning_module_sharded(wrapped_model) -> LightningModule:
             model = model.module
 
         return unwrap_lightning_module(model)
+
+
+LightningFullShardedDataParallel = None
+if _FAIRSCALE_FULL_SHARDED_AVAILABLE:
+    from fairscale.nn import FlattenParamsWrapper
+    from fairscale.nn.data_parallel import FullyShardedDataParallel
+
+    class LightningFullShardedDataParallel(_LightningModuleWrapperBase):
+        # Just do this for later docstrings
+        pass
```

Comment:
Can you move this one up? This must not live inside the if clause.

Reply:
I modified it let me know if it looks ok, i just copied the one you've made for normal Sharded DDP